.github/workflows/conformance.yml

```diff
@@ -550,7 +564,7 @@ jobs:
         set +e
         gotestsum --jsonfile ${{ env.TEST_OUTPUT_FILE_PREFIX }}_conformance.json \
           --junitfile ${{ env.TEST_OUTPUT_FILE_PREFIX }}_conformance.xml --format standard-verbose -- \
-          -p 2 -count=1 -timeout=15m -tags=conftests ./tests/conformance --run="Test${KIND_UPPER}Conformance/${NAME}"
+          -p 2 -count=1 -timeout=15m -tags=conftests ./tests/conformance --run="Test${KIND_UPPER}Conformance/${NAME}" -coverprofile=cover.out -coverpkg=github.com/dapr/components-contrib/${KIND}/${NAME}/...
```

Comment:
One thing to note is that some components have "shared" code located in a folder in the `internal/` package. Perhaps that should be included in the coverage?